{
	"header": {
		"line": "CVE-2022-3559 (A vulnerability was found in Exim and classified as problematic. This  ...)",
		"name": "CVE-2022-3559",
		"description": "(A vulnerability was found in Exim and classified as problematic. This  ...)"
	},
	"annotations": [
		{
			"line": "\t- exim4 4.96-4",
			"type": "package",
			"package": "exim4",
			"kind": "fixed",
			"version": "4.96-4"
		},
		{
			"line": "\t[buster] - exim4 <no-dsa> (Minor issue)",
			"type": "package",
			"release": "buster",
			"package": "exim4",
			"kind": "no-dsa",
			"description": "Minor issue"
		},
		{
			"line": "\tNOTE: https://bugs.exim.org/show_bug.cgi?id=2915",
			"type": "NOTE",
			"description": "https://bugs.exim.org/show_bug.cgi?id=2915"
		},
		{
			"line": "\tNOTE: https://git.exim.org/exim.git/commit/4e9ed49f8f12eb331b29bd5b6dc3693c520fddc2",
			"type": "NOTE",
			"description": "https://git.exim.org/exim.git/commit/4e9ed49f8f12eb331b29bd5b6dc3693c520fddc2"
		}
	]
}
